src/pages/LandingPage/LandingPage.tsx

```diff
@@ -6,13 +6,14 @@ import LoadableContent from 'src/components/LoadableContent';
 import Loader from 'src/components/Loader';
 import { Heading } from './styled';
 import { fetchEvents } from 'src/api/events';
+import { subHours } from 'date-fns';
 
 function LandingPage() {
   return (
     <section id="landing-page">
       <Heading>Events near you</Heading>
       <LoadableContent
-        resolveContent={fetchEvents}
+        resolveContent={() => fetchEvents(subHours(new Date(), 12))}
```

Comment:
Why subtract 12 hours? Do we want to show _some_ past events?

Reply:
Wrote it in the PR description, but organizers might update events after they are finished to include more links, information whatever so it should be possible to see past events. In later iterations there will be a checkbox or something to toggle to view past events. Should probably be able to search for events as well and get list for events where you participate. But this is good enough for now

Reply:
Ah sorry, forgot to read the description. But I see, that makes sense!